bin/standalone-installer-unix

```diff
@@ -27,7 +27,7 @@ declare KERNEL TARGET DESTINATION VERSION
 main() {
     KERNEL="${KERNEL:-$(uname -s)}"
     TARGET="${TARGET:-$(target-triple)}"
-    DESTINATION="${DESTINATION:-$(destination)}"
+    DESTINATION="${DESTINATION:-${NEXTSTRAIN_HOME:-$HOME/.nextstrain}/cli-standalone}"
```

Comment:
Since this isn't merged, it was a bit roundabout to test properly. This is what I did, in case anyone else finds it useful:

1. Follow the installation docs from the related PR https://github.com/nextstrain/docs.nextstrain.org/pull/123. The existing URL redirects to the installation script on the default branch. Replace the URL to install from this working branch:

    ```sh
    curl -fsSL --proto '=https' https://raw.githubusercontent.com/nextstrain/cli/trs/managed-conda-runtime/bin/standalone-installer-unix | bash

    ```

2. The installer will try to pull the latest released version. To test behavior from this PR, replace the installed files with artifacts from the latest CI run on the PR branch:

    ```sh
    runId=$(gh run list --repo nextstrain/cli --branch trs/managed-conda-runtime --workflow "CI" --json "databaseId" --jq ".[0].databaseId")
    # Linux
    gh run download --repo nextstrain/cli $runId --name standalone-x86_64-unknown-linux-gnu
    tar -xzf $(ls -1 nextstrain-cli-*-standalone-x86_64-unknown-linux-gnu.tar.gz) --directory ~/.nextstrain/cli-standalone
    # macOS
    gh run download --repo nextstrain/cli $runId --name standalone-x86_64-apple-darwin
    tar -xzf $(ls -1 nextstrain-cli-*-standalone-x86_64-apple-darwin.tar.gz) --directory ~/.nextstrain/cli-standalone
    ```

3. Run the setup commands from the output of (1).
4. Run `nextstrain setup managed-conda`
5. ...

Reply:
The installation script can optionally take a version (or `latest`, the default) to install, and I've been noodling with making it also accept something like `pr/218` or a GitHub Actions workflow run URL as an alternative. But it adds a bit of complication that I'm not sure is worth it unless we frequently find ourselves doing the song and dance above.

Reply:
> it adds a bit of complication that I'm not sure is worth it unless we frequently find ourselves doing the song and dance above.

Definitely, and I don't mind going through the current process frequently anyways.